### PR TITLE
DOCSP-33790 Deployment Considerations Content

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- Adding `deployment considerations` under `installation`.
- Adding `system-requirements` page under `installation`. 

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-33790/installation/system-requirements/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-33790/installation/deployment-considerations/

## JIRA

https://jira.mongodb.org/browse/DOCSP-39236
https://jira.mongodb.org/browse/DOCSP-33790

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66748d74ebce19f314a65727

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/deployment-considerations/colocation.txt [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+       hours.
+     - Remote server / EC2 / Azure VM
+
+Where to run Relational Migrator
+--------------------------------
+
+For best performance, locate the machine or server running Relational 
+Migrator as geographically close to the source database as possible: 
+
+- If your using Relational Migrator for an on-premises migration, this
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:43]: Wrong "your". This "your" is possessive. You want the contraction "you're".

File: source/deployment-considerations/colocation.txt [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+--------------------------------
+
+For best performance, locate the machine or server running Relational 
+Migrator as geographically close to the source database as possible: 
+
+- If your using Relational Migrator for an on-premises migration, this 
+  means running Relational Migrator in the same data center as the source
+  database.
+
+- If your using a cloud hosted database this means running Relational
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:47]: See above comments for this bullet.

File: source/deployment-considerations/colocation.txt [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+Where to run Relational Migrator
+--------------------------------
+
+For best performance, locate the machine or server running Relational 
+Migrator as geographically close to the source database as possible: 
+
+- If your using Relational Migrator for an on-premises migration, this 
+  means running Relational Migrator in the same data center as the source
+  database.
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:45]: This wording is a little clunky.

```suggestion
- If your using Relational Migrator for an on-premises migration, run Relational Migrator in the same data center as the source
  database.
```

File: source/deployment-considerations/colocation.txt [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+- If your using a cloud hosted database this means running Relational 
+  Migrator on an EC2 instance or VM in the same 
+  :abbr:`VPC (Virtual Private Cloud)` as the source database.
+
+.. tip::
+
+   Check the sleep timeout setting for your operating system. If your 
+   machine goes to sleep in the middle of a migration the sync job fails.
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:54]:
```suggestion
   machine goes to sleep during migration, the sync job fails. 
```

File: source/deployment-considerations/colocation.txt [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+Colocation
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+How to run Relational Migrator
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:13]: I think "Run" should be capitalized in these titles.

File: source/deployment-considerations/colocation.txt [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+==========
+Colocation
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:12]: I see that the page is titled colcation, but we don't define what that is anywhere on the page, and instead jump right into kind of more general recommendations for environments. Maybe we should add a brief introduction for what colocation is and why it's important?

File: source/deployment-considerations/colocation.txt [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+from the IPV4 address of the machine or server running Relational
+Migrator.
+
+VPC and Cloud Networking
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When running Relational Migrator on an AWS EC2 instance, check the 
+security group and firewall configurations of the EC2 instance or VM. 
+Also check that the router table allows traffic to the database. If your 
+database is hosted on a public IP address, this means that the EC2
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:71]:
```suggestion
database is hosted on a public IP address, the EC2 
```

File: source/deployment-considerations/colocation.txt [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+VPC and Cloud Networking
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When running Relational Migrator on an AWS EC2 instance, check the 
+security group and firewall configurations of the EC2 instance or VM. 
+Also check that the router table allows traffic to the database. If your 
+database is hosted on a public IP address, this means that the EC2 
+instance needs a router table entry to an internet gateway within the 
+:abbr:`VPC (Virtual Private Cloud)`. For details on specific cloud 
+provider network configurations see:
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:74]: Generally we want to introduce lists with a full sentence, so maybe we can try:

```suggestion
provider network configurations see these pages:
```

File: source/deployment-considerations/colocation.txt [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+- `AWS Networking <https://docs.aws.amazon.com/vpc/>`__
+- `Azure Networking <https://azure.microsoft.com/en-us/products/category/networking>`__
+- `GCP Networking <https://cloud.google.com/vpc/docs>`__
+
+AWS Private Link
+~~~~~~~~~~~~~~~~
+
+`AWS Private Link <https://docs.aws.amazon.com/vpc/latest/privatelink/what-is-privatelink.html>`__ 
+is a robust, scalable solution allowing you to
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:84]:
```suggestion
is a robust, scalable solution that allows you to 
```

File: source/deployment-considerations/colocation.txt [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+provider network configurations see:
+
+- `AWS Networking <https://docs.aws.amazon.com/vpc/>`__
+- `Azure Networking <https://azure.microsoft.com/en-us/products/category/networking>`__
+- `GCP Networking <https://cloud.google.com/vpc/docs>`__
+
+AWS Private Link
+~~~~~~~~~~~~~~~~
+
+`AWS Private Link <https://docs.aws.amazon.com/vpc/latest/privatelink/what-is-privatelink.html>`__
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:83]: Does this blurb need to be so marketing-y?

File: source/deployment-considerations/colocation.txt [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+- `GCP Networking <https://cloud.google.com/vpc/docs>`__
+
+AWS Private Link
+~~~~~~~~~~~~~~~~
+
+`AWS Private Link <https://docs.aws.amazon.com/vpc/latest/privatelink/what-is-privatelink.html>`__ 
+is a robust, scalable solution allowing you to 
+establish private connections between your VPC and various services, 
+making them feel as though they're located within your own VPC. This 
+eliminates the necessity for an internet gateway or public IP addresses.
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:87]: This last sentence is pretty clunky. What about:

"With AWS Private Link, you don't need an internet gateway or public IP addresses."

File: source/deployment-considerations/colocation.txt [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+
+You can use Relational Migrator with AWS private link. You need to 
+configure a
+`AWS private link endpoint <https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html>`__.
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:91]: Consider:

To use Relational Migrator with AWS Private Link, you need to configure...

File: source/installation/system-requirements.txt [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+The following page outlines the system requirements for using Relational 
+Migrator. For additional details on where to run Relation Migrator, see 
+:ref:`rm-colocation`.
+
+Hardware
+--------
+
+Computer (can be a local desktop/labtop or remote server), 
+Virtual Machine or Docker Container with at least 8GB of RAM 
+(16GB or more recommended).
```
Comment (jeff-allen-mongo) [inline source/installation/system-requirements.txt:22]: I'm a little confused here, are these three separate things? (Computer, VM, Docker Container)? Bullets might help?

File: source/installation/system-requirements.txt [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+
+Network
+-------
+
+Deploy Relational Migrator to a location close to the destination 
+databases to maximise bandwidth and minimise latency. For additional
```
Comment (jeff-allen-mongo) [inline source/installation/system-requirements.txt:28]:
```suggestion
databases to maximize bandwidth and minimize latency. For additional 
```

Comment (jeff-allen-mongo) [inline source/installation/system-requirements.txt:28]: Consider rewriting this sentence as "To (x), (do y)"

> To maximize bandwidth and minimize latency, deploy RM to...

File: source/installation/system-requirements.txt [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+Network
+-------
+
+Deploy Relational Migrator to a location close to the destination 
+databases to maximise bandwidth and minimise latency. For additional 
+details, see :ref:`rm-colocation`.
+
+Operating System
+----------------
+
```
Comment (jeff-allen-mongo) [inline source/installation/system-requirements.txt:33]: Consider introducing this list with something like:

"You can run Relational Migrator on one of the following operating systems: "

File: source/installation/system-requirements.txt [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+- Windows 10 or higher
+- Windows Server 2016 or higher
+- MacOS 11 or higher (Intel or ARM)
+- Linux (supported Debian or CentOS based distributions)
```
Comment (jeff-allen-mongo) [inline source/installation/system-requirements.txt:37]: Are only Debian and CentOS supported? This isn't totally clear to me.

File: source/installation/system-requirements.txt [hunk 17]
```diff
@@ -0,0 +1,55 @@
...
+- Docker, running on a host supporting the Docker client
+
+Web Browser
+-----------
+
+Current supported versions of Chrome (or Chromium-based browsers),
```
Comment (jeff-allen-mongo) [inline source/installation/system-requirements.txt:43]: Maybe use bullets here.

You can use Relational Migrator in one of the following web browsers:

- Current supported versions of Chrome (or Chromium-based browsers)
- Edge
- ...

File: source/installation/system-requirements.txt [hunk 18]
```diff
@@ -0,0 +1,55 @@
...
+Web Browser
+-----------
+
+Current supported versions of Chrome (or Chromium-based browsers), 
+Edge, Firefox or Safari.
+
+Kafka Deployments
+-----------------
+
+Optional component, recommended for long-running sync jobs. For details
```
Comment (jeff-allen-mongo) [inline source/installation/system-requirements.txt:49]: Not totally sure what we're saying is optional here. I think writing this as a more complete sentence would help.

"You can optionally use Kafka for long-running sync jobs. For details..."

File: source/installation/system-requirements.txt [hunk 19]
```diff
@@ -0,0 +1,55 @@
...
+see :ref:`kafka-intro`.
+
+Supported Databases
+-------------------
+
+:ref:`supported-databases`
```
Comment (jeff-allen-mongo) [inline source/installation/system-requirements.txt:55]:
```suggestion
See :ref:`supported-databases`.
```

**Review by jeff-allen-mongo — CHANGES_REQUESTED** [review]
@ianf-mongodb I left some suggestions. Lmk if you want to discuss.

**Review by ianf-mongodb — COMMENTED** [review]
Thanks for the review @jeff-allen-mongo, this is ready for another look.

File: source/deployment-considerations.txt [hunk 20]
```diff
@@ -0,0 +1,20 @@
...
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The pages in this section outline considerations that should be taken 
+into account as you prepare to deploy Relational Migrator.
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations.txt:14]: Can we shorten to:

The pages in this section help you prepare to deploy Relational Migrator.

?

File: source/deployment-considerations/colocation.txt (hunk 6 above)
Comment (jeff-allen-mongo) [reply source/deployment-considerations/colocation.txt:12]: Nice addition. The context is clearer now.

File: source/deployment-considerations/colocation.txt [hunk 21]
```diff
@@ -0,0 +1,105 @@
...
+
+In the context of Relational Migrator, colocation refers to where the 
+servers, networking equipment, and databases are located. This page 
+outlines location specific details to consider when using Relational 
+Migrator.
+
+How to Run Relational Migrator
+------------------------------
+
+The following table outlines some high level recommendations for how to
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:21]:
```suggestion
The following table outlines high-level recommendations for how to 
```

File: source/deployment-considerations/colocation.txt [hunk 22]
```diff
@@ -0,0 +1,105 @@
...
+
+   * - Smaller scale or debugging and testing jobs that run for 
+       less than four hours. 
+     - Laptop / Desktop / Docker
+
+   * - Larger scale production jobs that run for more than four 
+       hours.
+     - Remote server / EC2 / Azure VM
+
+Where to run Relational Migrator
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:42]:
```suggestion
Where to Run Relational Migrator
```

File: source/deployment-considerations/colocation.txt [hunk 23]
```diff
@@ -0,0 +1,105 @@
...
+
+For best performance, locate the machine or server running Relational 
+Migrator as geographically close to the source database as possible: 
+
+- If you're using Relational Migrator for an on-premises migration, 
+  run Relational Migrator in the same data center as the source 
+  database.
+
+- If you're using a cloud hosted database this means running Relational 
+  Migrator on an EC2 instance or VM in the same
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:53]:
```suggestion
- If you're using a cloud hosted database, run Relational 
  Migrator on an EC2 instance or VM in the same 
```

File: source/deployment-considerations/colocation.txt [hunk 24]
```diff
@@ -0,0 +1,105 @@
...
+
+- `AWS Networking <https://docs.aws.amazon.com/vpc/>`__
+- `Azure Networking <https://azure.microsoft.com/en-us/products/category/networking>`__
+- `GCP Networking <https://cloud.google.com/vpc/docs>`__
+
+AWS Private Link
+~~~~~~~~~~~~~~~~
+
+`AWS Private Link <https://docs.aws.amazon.com/vpc/latest/privatelink/what-is-privatelink.html>`__ 
+is a solution that allows you to establish private connections between
```
Comment (jeff-allen-mongo) [inline source/deployment-considerations/colocation.txt:89]:
```suggestion
is a solution that lets you establish private connections between 
```

File: source/deployment-considerations/colocation.txt (hunk 10 above)
Comment (jeff-allen-mongo) [reply source/deployment-considerations/colocation.txt:88]: Thanks, I think this is much better.

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM % a couple remaining nits. Thanks for the edits @ianf-mongodb

File: source/deployment-considerations/colocation.txt (hunk 22 above)
Comment (ianf-mongodb) [reply source/deployment-considerations/colocation.txt:42]: Woops sorry I missed this from the last round.

File: source/deployment-considerations.txt (hunk 20 above)
Comment (ianf-mongodb) [reply source/deployment-considerations.txt]: Updated.

**Comment by ianf-mongodb** [comment]
Hi @tomhollander this is ready for review and should cover 2 of the tickets in the RM docs backlog:
system requirements and colocation documentation.

File: source/deployment-considerations/colocation.txt (hunk 6 above)
Comment (tomhollander) [reply source/deployment-considerations/colocation.txt:12]: Do we really need a hierarchy here? "Deployment considerations" is a better page title than "Colocation".

File: source/deployment-considerations/colocation.txt [hunk 25]
```diff
@@ -0,0 +1,105 @@
...
+Colocation
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In the context of Relational Migrator, colocation refers to where the
```
Comment (tomhollander) [inline source/deployment-considerations/colocation.txt:13]: As above, I don't think this term is helpful. "Colocation" could imply running multiple things on the same machine, which we don't want to encourage.

File: source/deployment-considerations/colocation.txt [hunk 26]
```diff
@@ -0,0 +1,105 @@
...
+
+   * - Larger scale production jobs that run for more than four 
+       hours.
+     - Remote server / EC2 / Azure VM
+
+Where to Run Relational Migrator
+--------------------------------
+
+For best performance, locate the machine or server running Relational 
+Migrator as geographically close to the source database as possible:
```
Comment (tomhollander) [inline source/deployment-considerations/colocation.txt:46]: @alwaysbemark If someone is forced to choose between being close to the source DB and close to the destination DB, is one better than the other?

File: source/deployment-considerations/colocation.txt [hunk 27]
```diff
@@ -0,0 +1,105 @@
...
+.. tip::
+
+   Check the sleep timeout setting for your operating system. If your 
+   machine goes to sleep during a migration, the sync job fails. 
+
+Network Considerations
+----------------------
+
+When using Relational Migrator, the system firewall on the machine or 
+server must allow inbound and outbound TCP traffic to the source
```
Comment (tomhollander) [inline source/deployment-considerations/colocation.txt:65]: I don't think we need inbound? @alwaysbemark please confirm.

File: source/deployment-considerations/colocation.txt [hunk 28]
```diff
@@ -0,0 +1,105 @@
...
+database. The source database connection must also allow TCP traffic 
+from the IPV4 address of the machine or server running Relational
```
Comment (tomhollander) [inline source/deployment-considerations/colocation.txt:67]: Not sure we need to state IPv4. I'd imagine it's possible to get this working on IPv6 if you're keen. Just "IP address" to make it generic?

File: source/deployment-considerations/colocation.txt [hunk 29]
```diff
@@ -0,0 +1,105 @@
...
+Migrator.
+
+VPC and Cloud Networking
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When running Relational Migrator on an AWS EC2 instance, check the
```
Comment (tomhollander) [inline source/deployment-considerations/colocation.txt:73]: This advice applies to all cloud providers. Can you rewrite it so it doesn't use AWS-specific terminology?

File: source/deployment-considerations/colocation.txt [hunk 30]
```diff
@@ -0,0 +1,105 @@
...
+database is hosted on a public IP address, the EC2 
+instance needs a router table entry to an internet gateway within the 
+:abbr:`VPC (Virtual Private Cloud)`. For details on specific cloud 
+provider network configurations, see these pages:
+
+- `AWS Networking <https://docs.aws.amazon.com/vpc/>`__
+- `Azure Networking <https://azure.microsoft.com/en-us/products/category/networking>`__
+- `GCP Networking <https://cloud.google.com/vpc/docs>`__
+
+AWS Private Link
```
Comment (tomhollander) [inline source/deployment-considerations/colocation.txt:85]: I don't think this section adds anything. There isn't anything here specific to RM. There are also equivalent technologies for other cloud vendors.

File: source/deployment-considerations/colocation.txt [hunk 31]
```diff
@@ -0,0 +1,105 @@
...
+`AWS Private Link <https://docs.aws.amazon.com/vpc/latest/privatelink/what-is-privatelink.html>`__ 
+is a solution that lets you establish private connections between 
+your VPC and various services. AWS private link makes network traffic 
+behave as though the connected systems are located within your own VPC.
+
+You can configure a `AWS private link endpoint <https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html>`__ 
+for usage with Relational Migrator. This setup 
+eliminates the need for an internet gateway or public IP addresses.
+
+System Hardware
```
Comment (tomhollander) [inline source/deployment-considerations/colocation.txt:97]: Somewhere (not necessarily this section), can we call out that migration is I/O intensive on the destination MongoDB cluster, and that scaling up the cluster can significantly reduce migration time?

File: source/installation/system-requirements.txt [hunk 32]
```diff
@@ -0,0 +1,73 @@
...
+Migrator. For additional details on where to run Relation Migrator, see 
+:ref:`rm-colocation`.
+
+Hardware
+--------
+
+You can run Relational Migrator in multiple ways:
+
+- Computer (can be a local desktop/labtop)
+- Remote server (EC2, Azure VM, or Google compute engine instance)
```
Comment (tomhollander) [inline source/installation/system-requirements.txt:23]: Can we add a note that you must control RM through a web browser, which requires either:
 - Loading a browser on the machine running RM (locally or using Windows RDP or similar)
 - Accessing remotely, which requires configuring RM in Unattended mode

File: source/installation/system-requirements.txt [hunk 33]
```diff
@@ -0,0 +1,73 @@
...
+- Windows Server 2016 or higher
+- MacOS 11 or higher (Intel or ARM)
+- Linux - Debian / Ubuntu
+- Linux - Redhat / CentOS 
+- Docker
+
+Web Browser
+-----------
+
+You can use Relational Migrator in any of the following web browsers:
```
Comment (tomhollander) [inline source/installation/system-requirements.txt:57]: Should say "latest versions of"

File: source/installation/system-requirements.txt [hunk 34]
```diff
@@ -0,0 +1,73 @@
...
+Kafka Deployments
+-----------------
+
+You can optionally use Kafka for long-running sync jobs. For details.
+see :ref:`kafka-intro`.
+
+Supported Databases
+-------------------
+
+See :ref:`supported-databases`.
```
Comment (tomhollander) [inline source/installation/system-requirements.txt:73]: We should also link to the sync prerequisites page.

**Review by tomhollander — CHANGES_REQUESTED** [review]
Good start, added a few comments. 
I'd also like @alwaysbemark to review.

**Comment by ianf-mongodb** [comment]
Hey @alwaysbemark, could you have a look at this / @tomhollander questions when you get a moment? I will clarify based on your responses. Thanks!

File: source/installation/deployment-considerations.txt [hunk 35]
```diff
@@ -0,0 +1,111 @@
...
+
+   * - Larger scale production jobs that run for more than four 
+       hours.
+     - Remote server / EC2 / Azure VM
+
+Where to Run Relational Migrator
+--------------------------------
+
+For best performance, locate the machine or server running Relational 
+Migrator as geographically close to the source database as possible:
```
Comment (alwaysbemark) [inline source/installation/deployment-considerations.txt:46]: Should we say as close to the source and target, with the target bring a priority?

File: source/installation/deployment-considerations.txt [hunk 36]
```diff
@@ -0,0 +1,111 @@
...
+.. tip::
+
+   Check the sleep timeout setting for your operating system. If your 
+   machine goes to sleep during a migration, the sync job fails. 
+
+Network Considerations
+----------------------
+
+When using Relational Migrator, the system firewall on the machine or 
+server must allow inbound and outbound TCP traffic to the source
```
Comment (alwaysbemark) [inline source/installation/deployment-considerations.txt:65]: We'll need outbound only for both source and target.

File: source/installation/deployment-considerations.txt [hunk 37]
```diff
@@ -0,0 +1,111 @@
...
+database is hosted on a public IP address, the EC2 
+instance needs a router table entry to an internet gateway within the 
+:abbr:`VPC (Virtual Private Cloud)`. For details on specific cloud 
+provider network configurations, see these pages:
+
+- `AWS Networking <https://docs.aws.amazon.com/vpc/>`__
+- `Azure Networking <https://azure.microsoft.com/en-us/products/category/networking>`__
+- `GCP Networking <https://cloud.google.com/vpc/docs>`__
+
+AWS Private Link
```
Comment (alwaysbemark) [inline source/installation/deployment-considerations.txt:85]: I'm not sure we have explored this enough to recommend on the official docs. We haven't had too many requests come in for this (yet) so I'd be ore comfortable leaving PrivateLink (and any specific networking solutions) out for now if that's OK!

**Review by ianf-mongodb — COMMENTED** [review]
Hi @tomhollander @alwaysbemark this is ready for another look, I have updated per your previous feed back.

**Review by tomhollander — APPROVED** [review]

**Review by alwaysbemark — APPROVED** [review]
Nice one, thanks Ian!

**Pull request merged**